### PR TITLE
[11.0][IMP] Add manual attendance group

### PR DESCRIPTION
**Pull request by jarroyomorales**

### CODE REVIEW

File: cb_hr_views/security/hr_attendance_security.xml [hunk 1]
```diff
@@ -8,6 +8,10 @@
         <field name="name">Employee</field>
     </record>
 
+    <record id="manual_attendance_group" model="res.groups">
+        <field name="name">Manual Attendance</field>
```
Comment (etobella) [inline cb_hr_views/security/hr_attendance_security.xml]: Change for Manual Attendance Menu please

**Review by etobella — CHANGES_REQUESTED** [review]

File: cb_hr_views/security/hr_attendance_security.xml (hunk 1 above)
Comment (jarroyomorales) [reply cb_hr_views/security/hr_attendance_security.xml]: Changed!

**Comment by jarroyomorales** [comment]
@etobella :green_apple: :green_apple: :green_apple: :green_apple:

File: cb_hr_views/views/hr_attendance_module_views.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <record model="ir.ui.menu" id="hr_attendance.menu_hr_attendance_my_attendances">
-        <field name="groups_id" eval="[(6, 0, [ref('make_invisible')])]"/>
+        <field name="groups_id" eval="[(6, 0, [ref('manual_attendance_group')])]"/>
     </record>
     <record model="ir.ui.menu" id="hr_attendance.menu_hr_attendance_kiosk_mode">
         <field name="groups_id" eval="[(6, 0, [ref('make_invisible')])]"/>
```
Comment (etobella) [inline cb_hr_views/views/hr_attendance_module_views.xml]: Remove make_invisible and set active=False

**Review by etobella — CHANGES_REQUESTED** [review]

File: cb_hr_views/views/hr_attendance_module_views.xml (hunk 2 above)
Comment (etobella) [reply cb_hr_views/views/hr_attendance_module_views.xml]: @jarroyomorales

Comment (jarroyomorales) [reply cb_hr_views/views/hr_attendance_module_views.xml]: Se me olvidó hacer push

File: cb_hr_views/views/hr_attendance_module_views.xml [hunk 3]
```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <record model="ir.ui.menu" id="hr_attendance.menu_hr_attendance_my_attendances">
-        <field name="groups_id" eval="[(6, 0, [ref('make_invisible')])]"/>
+        <field name="groups_id" eval="[(6, 0, [ref('manual_attendance_group')])]"/>
```
Comment (etobella) [inline cb_hr_views/views/hr_attendance_module_views.xml:4]: Remove the make_invisible group as it is  no longer necessary

**Review by etobella — CHANGES_REQUESTED** [review]

**Review by etobella — APPROVED** [review]

**Pull request merged**